feat(content): Successors: Ghosts I (#12221)
endless-sky/endless-sky: data/successors/successor 2 ghosts myurej.txt

			`	You are escorted back to the edge of the residential area by the guards, who leave you to find your own way back to your ship. As the ground crew finish up their refueling and maintenance of the <ship>, one of them hands you a data-sphere containing information about Riiria and House Myurej's contact on Kasii-Tuur-Saqru.`
				accept

			label run
			`	Kicking a stone to the side to distract the pair, you dash into the crowd before they can stop you. It's probably lucky that they didn't shoot you for that.`
				decline
			label no
			`	The voice gives a long hum but says no more. After a couple moments you hesitantly get up and, when the guards make no motion to prevent you, you leave.`
				decline
	on complete
		conversation
			`You guide the <ship> into the crowded spaceport of <planet>, narrowly avoiding a collision with one of a wing of armored-up Kvelqden rocketing out of the docking section.`
			choice
				`	(Seek out Myurej's contact.)`
					goto contact
				`	(Look around for yourself first.)`
			action
				"successors: ghosts suspicion" ++

> 			`	Exploring the visitor-accessible part of the station is an unusual experience - there's a striking contrast between the presence of all the advanced technologies of the Successors and the fact that so many of them are in such disrepair. You've seen Successor constructions that are hundreds or even thousands of years old in better condition than those here. You try to talk to a few of the Successors around here about Riiria, but they all quickly dismiss you, most without even saying a word.`

Reviewer: > You try to talk to a few of the Successors around here about Riiria,

This part is a bit confusing. I just chose `(Look around for yourself first.)` above. Should it be re-worded somehow?

			`	"You're going to have to give me more than that," you say.`
			`	The wave rebounds off the far wall and retraces its way back down the screen. "Hast thou before entered into the bonds of matrimony?" the voice says. Before you can respond, it continues. "'Tis customary, when one doth, for the celebrants to render some minor gifts upon the duly unified, a commemoration of the families' intertwining evermore. To steal such trinkets would be a base and wretched act - a child-race as yours should still conceive of that." The wave continues to echo, back and forth. "A certain knave'f the so-called People's Houses thought it wise to thus insult this happy House, on the eve of another day of joy, no doubt thinking with a dead mind they lay beyond our reach. Bring back our gifts, and we shall pay thee..." The voice pauses for a while, as if away in thought. "And perhaps thou wilt have earned a favor, though alien thou might'st be."`
			choice
				`	"Why ask me?"`
				`	"How much are you going to pay me?"`
					goto pay
				`	"All right, I'll do it."`
					goto yes
				`	"Sorry, but I don't want any part of this."`
					goto no
			branch kaatrij

> 				has "successors kaatrij patron"

Reviewer 2: Should be sioeora patron, no? (looking at line right below)

Author: Branches work by jumping to the `label` that has the same name as the `branch` (here 'kaatrij') if their condition is correct, skipping over lines in between the branch and the corresponding label. So I think this instance should work correctly.

			`	"Riijaaora of House Sioeora speaks well of thee, dost thou know? Thy reliability, thy strength of will. They say thou'rt capable of this task, and we know thou'rt uniquely suited to it. Shalt thou accept?"`
				goto "follow up"
			label kaatrij
			`	"After thou fail'd Sioeora as thou did'st, thinks thee? Nonetheless, thou'rt more suitable to our needs than any other. Shoulds't thou decline, we shall simply move on to the next. So, shalt thou accept?"`
				goto "follow up"
			label pay

> 			`	The voice hums briefly. "four hundred thousand credits," they say, "not to be negotiated. Dost thou accept?"`

Reviewer 3: ```suggestion
			`	The voice hums briefly. "Four hundred thousand credits," they say, "not to be negotiated. Dost thou accept?"`
```

			label contact
			`	The Myurej merchant captain turns out to be almost a kilometer down the station's perimeter among some loading docks that station control apparently doesn't send unfamiliar ships to. They're directing several members of their crew to offload some crates - one of them has been dropped and has spilled out a collection of unfamiliar metal baubles.`
			choice
				`	"I'm Captain <first> <last>. Myurej sent me."`
					goto introduce
				`	"What are those things?"`
			`	"What cares thou? Thou'rt in the way, go off." They wave you away.`
			`	"Fine," you say. "Captain <first> <last>. Myurej sent me."`
			label introduce
			`	A wave of red and black bristles over their skin. "Of course," they say. "I should've guessed. Come to send me on some wild quest for some bit of finery, is it? Some golden child lost their favorite jewel?"`

> 			`	"Well, sort of," you say. "All I need from you is the location of Riiria. They stole something from the House that they want back."`

Reviewer 3: Lots of theys.
```suggestion
			`	"Well, sort of," you say. "All I need from you is the location of Riiria. They stole something from the House that the House wants back."`
```

			choice
				`	(Rummage through the pack.)`
				`	(Prepare to leave.)`
					goto end
			action
				"successors: ghosts suspicion" ++
			`	Inside you find, among the odds and ends of this Successor's belongings, a still-warm data-sphere. It springs to life as you touch it, relaying its most recent message:`
			`	"Riiria says the Houses shan't find us here, that they see these wretched hulks as icons'f their folly they daren't touch. They're right, I know, and yet I shudder most at thinking of that alien the Houses're so fond'f using. Riirira says fools're much more fearsome than the wise."`
			`	The message ends there, and you can't quite figure out how to access the rest of the data on the sphere, so you leave it be.`
			label end
			`	If Riiria isn't at this station, they might be at the other one to the south, so you head back to your ship and enter the coordinates on your map.`

mission "Successors: Ghosts Myurej 3"
	landing
	name "Continue the search"
	description "Having not found Riiria at <origin>, look for them at <destination>."
	source Vade-Osolaa-Kaska
	destination Myiara-Sola-Tej
	to offer
		has "Successors: Ghosts Myurej 2: done"

> 	on complete

Reviewer 3: I don't quite see how this mission gets completed?

		conversation
			`Not longer after you guide the <ship> through this station's docking aperture, you see parked on one of the disused landing pads a clearly unpowered Successor freighter. It's in poor condition; much of the hull armor has been removed and a hole has been bored down through the top right over the generator bay.`
			`	Fortunately, your ship's scanners are still able to identify its identification number, and you scrabble for House Myurej's data-sphere to check it against the data Myurej provided you. This is the ship you were sent to find, you determine.`
			choice
				`	(Use your cargo scanner to identify its cargo.)`
					to display
						has "flagship attribute: cargo scan power"
				`	(Check its cargo yourself.)`
					goto self
			`	Unsurprisingly, the ship's cargo bays have been picked clean and whatever wedding gifts you have been sent to retrieve aren't here. The scanner is able to interface with an unshielded, barely-powered terminal inside the hulk, however, and relays to you the ship's most recent cargo manifest:`
			``
				goto manifest
			label self

> 			`	You disembark the <ship> and walk into the ripped-open cargo bay of the Ojde to confirm what already seems likely: any trace of the ship's cargo has been removed. You are able to access an independently-powered terminal inside the bay, however, and as it burbles to life, it shows you the ship's most recent cargo manifest:`

Reviewer 3: This feels somewhat more natural.

```suggestion
			`	You disembark the <ship> and walk into the ripped-open cargo bay of the Ojde to confirm what already seems likely: every trace of the ship's cargo has been removed. You are able to access an independently-powered terminal inside the bay, however, and as it burbles to life, it shows you the ship's most recent cargo manifest:`
```

			label more
			`	"Like before," the voice says, "when Sioeora asked their favor of thee, we've aught we wish to reclaim from the gyre of the Shroud. Nothing to upset thy delicate ethics - naught but trinkets from when the House was young. 'Twould make a fair consolation to us, and we would forgive thee, and thou would'st know the beneficence of this House."`
			branch "not met solitude enforcers"
				not "Successors: Ghosts Aqrabe 7: offered"
			`	"Certain some drone warship isn't going to come after me as soon as I grab your trinkets?" you say, recalling the experience with House Aqrabe. This seems to catch the voice off guard, if only a little, and it takes a moment before it replies. "Any encounter thou hast met once before, thou should'st surely be unafraid of facing again."`
			label "not met solitude enforcers"
			choice
				`	"I don't appreciate being treated as disposable."`
					to display
						has "Successors: Ghosts Aqrabe 7: offered"
				`	"All right, I'll do it. But you'd better hold up your end of the bargain."`
					goto yes
				`	"I'm done with this."`
					goto no

> 			`	"Thou'rt not a tool to be thrown away, Captain, but to be cultivated, polished to a gleam. We would not send thee to thy death - how benefits the House in that? Return to the spaceport here when thou'rt ready to begin thy task."`

Reviewer 3: Slightly more natural.

```suggestion
			`	"Thou'rt not a tool to be thrown away, Captain, but to be cultivated, polished to a gleam. We would not send thee to thy death - what benefits the House from that? Return to the spaceport here when thou'rt ready to begin thy task."`
```

				launch
			label run
			`	You run back through the field of columns as the bombardment continues. Some of the impacts feel like they're getting closer the farther you run, but eventually you make it back to where you landed the <ship> and hurry inside with Myurej's agents right behind you. As you make your way to the bridge, another impact strikes dangerously close to your landing area, and you prepare to fly into orbit while you still can.`
				launch

mission "Successors: Ghosts Myurej 6"
	landing
	priority
	name "Flee back to Successor space"
	description "Escape the warships attacking you and regroup in Successor space."
	source Kella-Uuoru-Sossa
	destination Uuoru-Veldt-Stir
	passengers 3
	to offer
		has "Successors: Ghosts Myurej 5: done"
	npc evade
		ship "Queru, Luminous Gaze" "Kis, Lamenting 9-3"
		ship "Queru, Luminous Gaze" "Maki, Recalling 11-2"
		government "Solitude Enforcer"
		personality waiting heroic nemesis unconstrained vindictive
		on kill
			set "Successors: Ghosts Queru transmission"

> 			dialog "As the second drone ship explodes it transmits some data to your own vessel. It cannot be deciphered at this time."

Reviewer 3: Extremely minor, but the "own" felt very slightly out of place here.

```suggestion
			dialog "As the second drone ship explodes it transmits some data to your vessel. It cannot be deciphered at this time."
```

			`	Eventually the elevator nestles into a complex on the seafloor, and the two guards guide you through a number of transparent halls to a large central area. Fish swim above you, schooling behind a circular oculus. Another Successor approaches from one of the side corridors, and it is immediately apparent that this is one of Myurej's scions - they are draped in enough red gold and jewels that it's a wonder they can hold their body upright. When they speak, it's not the voice you heard behind the waving screen before, but one that's smoother and more inquisitive.`
			`	"Captain, captain," they say, "thou, Captain, who has brought us such lovely glitterings with such fine loyalty. And through such perilous straits, I hear as well? The knave Riiria, the fearsome guardians of the Shroud... I could not bear to keep myself from seeing thee in the flesh, the flesh that holds all this surpassing strength!" Their adornments rattle as they gesture, prodding the air gently.`
			choice
				`	"I hope I live up to your expectations."`
					goto flirty
				`	"It seems like a lot of effort for some little trinkets."`
					goto unamused
				`	"And you are?"`

> 			`	The scion fans their arms gently. "Myurej M- Khiajsore. A cousin of the one who spoke to thee earlier, whom thou shalt recall."`

Reviewer 3: I assume the M- is intentional?

				decline
			label no
			`	The voice gives a long hum but says no more. After a couple moments you hesitantly get up and, when the guards make no motion to prevent you, you leave.`
				decline
	on complete
		conversation
			`You guide the <ship> into the crowded spaceport of <planet>, narrowly avoiding a collision with one of a wing of armored-up Kvelqden rocketing out of the docking section.`
			choice
				`	(Seek out Myurej's contact.)`
					goto contact
				`	(Look around for yourself first.)`
			action
				"successors: ghosts suspicion" ++
			`	Exploring the visitor-accessible part of the station is an unusual experience - there's a striking contrast between the presence of all the advanced technologies of the Successors and the fact that so many of them are in such disrepair. You've seen Successor constructions that are hundreds or even thousands of years old in better condition than those here. You try to talk to a few of the Successors around here about Riiria, but they all quickly dismiss you, most without even saying a word.`

> 			`	Eventually you find your way to a shop of some sort, what looks like a gutted Myaede bolted to the station's structure and filled with displays of fabric and ship parts, all attended to by about a dozen small robots running on tracks on the ceiling. They follow you around like a school of fish - a couple of them have some ominous-looking apertures on their front. Suddenly, you hear the edited sound of Successor speech behind one of the displays of what looks like engine injectors. A small screen is projecting an image of a speaking Successor:`

Reviewer 4: ```suggestion
			`	Eventually you find your way to a shop of some sort - what looks like a gutted Myaede bolted to the station's structure and filled with displays of fabric and ship parts, all attended to by about a dozen small robots running on tracks on the ceiling. They follow you around like a school of fish; a couple of them have some ominous-looking apertures on their front. Suddenly, you hear the edited sound of Successor speech behind one of the displays of what looks like engine injectors. A small screen is projecting an image of a speaking Successor:`
```

			`	"Riiria says the Houses shan't find us here, that they see these wretched hulks as icons'f their folly they daren't touch. They're right, I know, and yet I shudder most at thinking of that alien the Houses're so fond'f using. Riiria says fools're much more fearsome than the wise."`
			`	The message ends there, and you can't quite figure out how to access the rest of the data on the sphere, so you leave it be.`
			label end
			`	If Riiria isn't at this station, they might be at the other one to the south, so you head back to your ship and enter the coordinates on your map.`

mission "Successors: Ghosts Myurej 3"
	landing
	name "Continue the search"
	description "Having not found Riiria at <origin>, look for them at <destination>."
	source Vade-Osolaa-Kaska
	destination Myiara-Sola-Tej
	to offer
		has "Successors: Ghosts Myurej 2: done"
	on complete
		conversation
			`Not longer after you guide the <ship> through this station's docking aperture, you see parked on one of the disused landing pads a clearly unpowered Successor freighter. It's in poor condition; much of the hull armor has been removed and a hole has been bored down through the top right over the generator bay.`

> 			`	Fortunately, your ship's scanners are still able to identify its identification number, and you scrabble for House Myurej's data-sphere to check it against the data Myurej provided you. This is the ship you were sent to find, you determine.`

Reviewer 4: ```suggestion
			`	Fortunately, your ship's scanners are still able to identify its identification number, and you scramble for House Myurej's data-sphere to check it against the data Myurej provided you. This is the ship you were sent to find, you determine.`
```

			label sioeora
			choice
				`	"I had nothing to do with Riiria's ship exploding."`
				`	"It seems to me like you're the one who needs to apologize. I was nearly killed!"`
					goto angry
				`	"I'm sorry."`
					goto grovel
			`	The screen ripples colorlessly. "What careth we? If we had sent another, would it still have been so?"`
				goto again
			label angry
			`	The screen ripples, dark and menacing. "Taking missions as these is what thou dost, is it not so? Is it common among lesser aliens as thee to berate one's clients for daring t'offer employ?"`
				goto again
			label grovel
			`	The screen ripples, lightless as tar. "We care little for thy apologies," says the voice, "but 'tis good that thou hast some cognizance of thine inability."`
			label again
			`	Suddenly, a wave of color springs across the screen. When the voice continues, there's something in it you can't quite place. "Thou couldst undo this failure for us, though, Captain. Find us some replacement for the things thou hast lost in thy folly. Thou couldst still earn our coin, our gratitude. Surely thou shouldst leap at this."`

> 			choice

Reviewer 4: Just a suggestion, but I think it would be fun to be able to mention that Myurej must not think that first last is that incompetent if they're offering them another job.

			`	"Fine," you say, "just tell me what you want so I can decide if I want any more to do with you."`
			label more
			`	"Like before," the voice says, "when Sioeora asked their favor of thee, we've aught we wish to reclaim from the gyre of the Shroud. Nothing to upset thy delicate ethics - naught but trinkets from when the House was young. 'Twould make a fair consolation to us, and we would forgive thee, and thou wouldst know the beneficence of this House."`
			branch "not met solitude enforcers"
				not "Successors: Ghosts Aqrabe 7: offered"
			`	"Certain some drone warship isn't going to come after me as soon as I grab your trinkets?" you say, recalling the experience with House Aqrabe. This seems to catch the voice off guard, if only a little, and it takes a moment before it replies. "Any encounter thou hast met once before, thou shouldst surely be unafraid of facing again."`
			label "not met solitude enforcers"
			choice
				`	"I don't appreciate being treated as disposable."`
					to display

> 						has "Successors: Ghosts Aqrabe 7: offered"

Reviewer 4: Wouldn't this choice never offer given the label `"not met solitude enforcers"` requires that you not have `Successors: Ghosts Aqrabe 7: offered`?

Author: The way I have structured it is a little confusing, but the `"not met solitude enforcers"` branch just skips over the player's little retort ("Certain some drone...") and takes them straight to the block of choices. So if the player has `"not met solitude enforcers"` they won't see the retort and they won't get the follow-up related to it.

			`	Eventually, Ijaset stops on the edge of one of the depressions. "Tis here," they say, emotionless, and point directions to the two guards. They check something against a datapad they produce from their clothing, and then glide over to one of the columns, extending three arms to feel around the circumference of it. Shortly after, something clicks, and a rumbling suddenly starts below you, growing louder and louder until it stops and an aperture opens in the middle of the depression. A tunnel yawns deep into the ground.`
			choice
				`	"That's a long way down."`
					goto down
				`	"Why put a vault in a place like this?"`
			`	Ijaset hums. "'Twas once a reserved quarter for the highborn of Myurej, this. Secret to those outside, but known to those who weren't. Then, as now, knowledge was a matter of position."`
			label down
			`	You go to look over the edge, and Ijaset joins you. "We should expect a long journey," they say. "'Twas meant to be shielded from just such a bombardment as came to pass."`

> 			`	The four of you make your way down the tunnel, which seems to go down into the planet for kilometers. The dark metal walls grow warm to the touch, and the slippery floor proves more treacherous for you than for the Successors - you have to catch yourself on the rail several times. Eventually you come to an enormous spherical room, large enough that the far wall fades into darkness even as a shaft above you lets in an impossibly-penetrating beam of sunlight. And ringing the walls are thousands upon thousands of circular insets, each one many meters in diameter, each one with a panel on the front taking - no doubt - a unique data-sphere.`

Reviewer 4: Not sure what `taking... a unique data sphere` means here?

			choice
				`	"I had no idea the Houses had this much."`
				`	"We can't be here just for jewelry, can we?"`
			`	"Myurej had dozens of these vaults," Ijaset says. "Most not here, not on the homeworld. Ships, weapons, great riches grown in the heart of stars... all should be preserved, lest the Quarg or some other despoilers have come to empty them. Today we come for trinkets, so tomorrow we can come for the rest."`
			choice
				`	"Let's just focus on what we're here for."`
					goto next

> 				`	"With the contents of a vault like this, any of the Houses could dominate the others, couldn't it?"`

Reviewer 4: ```suggestion
				`	"With the contents of a vault like this, any of the Houses could dominate the others, couldn't they?"`
```

	destination Iyra-Ijasa-Iret
	cargo "Wedding supplies" 5
	to offer
		has "Successors: Ghosts Myurej 7: done"
	on offer
		conversation
			`Once again traffic control directs you to the reserved portion of the <origin> spaceport with the bronze-trimmed landing pads. You are met outside by a somewhat stressed-looking dockworker who brightens up considerably when they see you.`
			`	"Of course, of course," they say, "we've been waiting for thee! A House transport to <planet> of supplies for the wedding, yes?"`
			choice
				`	"Excuse me?"`
				`	"I was suppose to meet someone here, I think."`
			`	The worker checks their datapad, looking back and forth between it and you before they once again announce with confidence that you are supposed to transport <tons> of wedding supplies to <planet> for the upcoming Myurej wedding.`
			choice
				`	"Well, get them loaded, I suppose."`
					accept
				`	"I'll come back later."`
					defer
	on complete
		"successors: ghosts houses dealt with" ++
		set "successors: ghosts dealt with myurej"
		log "Spoke with Khiajsore, a scion of House Myurej, at an underwater venue where an inter-House wedding is set to occur."
		conversation

> 			`The <ship> settles down in one of the metal bubbles of the <planet> spaceport and before long, as the ground crew is unloading your ship's cargo, a pair of House Myurej's soldiers comes to lead you through the facility. They take you to an elevator outside of the spaceport's public area; you step in, and a few seconds after it begins moving, it clears the bulk of the spaceport complex and shows you a fantastic view of the planet's ocean as it descends deeper into the water. Unfortunately you're on the wrong side of the planet to see the "Ribbons of Iret," but even the relatively dark sea is impressive in its scale and in the vastness of the underwater infrastructure the Successors have built within it.`

Reviewer 4: How would you know what the Ribbons of Iret are at this point?

Author: They are mentioned in the planet description. A little explanation (reminder) here in the text wouldn't go astray, though.
